src/xpcc/driver/can/mcp2515_impl.hpp

```diff
@@ -423,3 +437,17 @@ xpcc::Mcp2515<SPI, CS, INT>::readIdentifier(uint32_t& identifier)
 		return false;
 	}
 }
+
+xpcc::Abandonment can_assertion_handler(char *module, char *, char *, uintptr_t)
+{
+    if (!strncmp(module,   "mcp2515",  7) and
+    	!strncmp(location, "init",     4) and
+    	!strncmp(failure,  "readback", 8))
+    {
+        return xpcc::Abandonment::Fail;
+    }
+    return xpcc::Abandonment::DontCare;
+}
+
+// Register assertion handler with system
+XPCC_ASSERTION_HANDLER(can_assertion_handler);
```

Comment:
And when he/she forgets to define one? How does she/he know about the necessity to define one? Oh, I see

> There is no way of knowing if the function you’re calling can fail an assert [...]

Hmm, only a mother could love that?

Any chance to get a weak default handler?